docs/datasources/global_client.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "auth0"
+page_title: "Data Source: auth0_global_client"
+description: |-
+Retrieves a tenant's global Auth0 Application client
+---
+
+# Data Source: auth0_global_client
+
+Retrieves a tenant's global Auth0 Application client
+
+## Example Usage
+
+```hcl
+data "auth0_global_client" "global" {
+}
+```
+
+## Argument Reference
+
+No arguments accepted.
+
+## Attribute Reference
+
+* `client_id` - String. ID of the client.
+* `client_secret`<sup>[1](#client-keys)</sup> - String. Secret for the client; keep this private.
+* `custom_login_page_on` - Boolean. Indicates whether or not a custom login page is to be used.
+* `custom_login_page` - String. Content of the custom login page.
+* `client_metadata` - (Optional) Map(String)
```

Comment:
These lines are pulled directly from #363 , but my question is, why _only_ these lines are pulled if this shares the regular client schema. My suspicion is that it serves as deterrent for accessing other values which may not have value in the context of a global client, but I need to confirm.

Reply:
Learned that the global client's main use case is to manage the custom login page for classic login mode (legacy).